ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,18 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+MAZTM:
```

Comment:
What does the MA stand for? Should be ZTC rather than ZTM since you have setpoints (controlling rather than monitoring).

Reply:
I didnt think it was quite controlling though, the setpoints are to set the temperature at which the alarm is triggered?

Reply:
If you are including the alarm points themselves there is no need to include the thresholds at which the alarms are triggered.

Reply:
okay closing, thank you